controllers/jobs.js

```diff
@@ -0,0 +1,33 @@
+const Job = require('../models/Job');
+const { StatusCodes } = require('http-status-codes');
+const { BadRequestError, NotFoundError } = require('../errors');
+
+const getAllJobs = async (req, res) => {
+  res.send('Get all jobs');
+};
+
+const getJob = async (req, res) => {
+  res.send('Get all jobs');
+};
+
+const createJob = async (req, res) => {
+  req.body.createdBy = req.user.userId;
+  const job = await Job.create(req.body);
+  res.status(StatusCodes.CREATED).json({ job });
+};
+
+const updateJob = async (req, res) => {
+  res.send('Get all jobs');
+};
+
+const deleteJob = async (req, res) => {
+  res.send('Get all jobs');
```

Comment:
I know these are just placeholders but for the sake of making it easier to follow when you're testing things in Postman; it's good to use different text per route:

```suggestion
  res.send('Update a job');
};

const deleteJob = async (req, res) => {
  res.send('Delete a job');
```